src/functions-and-arrays.js

```diff
@@ -1,19 +1,86 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers (num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  } else if (num2 > num1) {
+    return num2;
+    } else {
+      return num1, num2;
+    }
```

Comment:
the indentation of the "else" statement should be on the same level as "if" and "else if"

Reply:
You don't actually need three statements here. Since, it does not matter which number you return if they are equal, you can handle this with if ... else ...